packages/mui-base/src/Popover/index.ts

```diff
@@ -0,0 +1,47 @@
+export { PopoverRoot as Root } from './Root/PopoverRoot';
+export { PopoverTrigger as Trigger } from './Trigger/PopoverTrigger';
+export { PopoverPositioner as Positioner } from './Positioner/PopoverPositioner';
+export { PopoverPopup as Popup } from './Popup/PopoverPopup';
+export { PopoverArrow as Arrow } from './Arrow/PopoverArrow';
+export { PopoverProvider as Provider } from './Provider/PopoverProvider';
+export { PopoverBackdrop as Backdrop } from './Backdrop/PopoverBackdrop';
+export { PopoverTitle as Title } from './Title/PopoverTitle';
+export { PopoverClose as Close } from './Close/PopoverClose';
+export { usePopoverRoot as useRoot } from './Root/usePopoverRoot';
+export { usePopoverPositioner as usePositioner } from './Positioner/usePopoverPositioner';
+
```

Comment:
So far, we haven't been renaming hooks (not sure if we should or not, but let's be consistent).

Reply:
I did do it for Tooltip I believe, but I can change that later to match as well